custom-dashboards/.openshift/patches/openshift-state-metrics-service-monitor.yaml

```diff
@@ -0,0 +1,6 @@
+spec:
```

Comment:
Probably should start yaml files with `---`

Reply:
@oybed added this to all yaml files, except for the ones for openshift-state-metrics as they were copied from another repo temporarily.